packages/zostso/src/IssueTso.ts

```diff
@@ -110,6 +110,13 @@ export class IssueTso {
             });
         }
 
+        if(response.startResponse?.messages?.includes("IKJ56482I")) {
+            throw new ImperativeError({
+                msg: `Invalid logon procedure.`,
+                additionalDetails: response.startResponse?.messages
```

Comment:
This is a nitpick, but sometimes our static code analysis tools are picky 🙂  We know that `response.startResponse` is defined once we're inside the `if` block.
```suggestion
                additionalDetails: response.startResponse.messages
```